src/Widget/LinkPager.php

```diff
@@ -153,6 +153,22 @@ private function setOption(string $name, $value): self
         return $new;
     }
 
+    /**
+     * Set option[class] for widget param
+     *
+     * @param string $name
+     * @param string $className
+     *
+     * @return self
+     */
+    private function setClassToOption(string $name, string $className): self
+    {
+        $value = $this->{$name};
```

Comment:
Let's avoid such magic. It's better to explicitly name fields with some duplication since it hurts both IDE and static analysis.

Reply:
> Let's avoid such magic. It's better to explicitly name fields with some duplication since it hurts both IDE and static analysis.

Ok, i close this PR and recreate new. I have case when doesn't fit bootstrap or bulma markups - and i have idea how do it. But now i have a broken right (main) arm and I do everything very slowly :)

Reply:
> Ok, i close this PR and recreate new.

No need to do it. Push additional commit to the same branch.

Reply:
> > Ok, i close this PR and recreate new.
> 
> No need to do it. Push additional commit to the same branch.

Done. 5hat was one psalm error, but i don't now how solve it